<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPropertiesDialog</class>
 <widget class="QDialog" name="ProjectPropertiesDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Project Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="projectNameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Main Source:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="mainSourceComboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Cartridge Info</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Mapper:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="mapperComboBox">
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="romTypeComboBox">
         <item>
          <property name="text">
           <string>NES</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VS Unisystem</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Playchoice</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Mirroring:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="mirroringComboBox">
         <item>
          <property name="text">
           <string>Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vertical</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="trainerPresent">
         <property name="text">
          <string>512-byte Trainer Present</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="saveRAMPresent">
         <property name="text">
          <string>Save RAM Present</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="extraVRAMPresent">
         <property name="text">
          <string>Extra VRAM Present</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>System Palette</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>15</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>15</number>
         </attribute>
         <row>
          <property name="text">
           <string>0x</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>1x</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2x</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3x</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>x0</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x5</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x6</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x7</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x8</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x9</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>xA</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>xB</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>xC</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>xD</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>xE</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>xF</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Emphasize Red</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="colorValueLabel">
         <property name="text">
          <string>#000000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="exportPalettePushButton">
         <property name="text">
          <string>Export Palette...</string>
         </property>
         <property name="icon">
          <iconset resource="../../resource.qrc">
           <normaloff>:/resources/document-export.svg</normaloff>:/resources/document-export.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="ImportPalettePushButton">
         <property name="text">
          <string>Import Palette...</string>
         </property>
         <property name="icon">
          <iconset resource="../../resource.qrc">
           <normaloff>:/resources/document-import.svg</normaloff>:/resources/document-import.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Green</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Red</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>Emphasize Blue</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>Emphasize Green</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Monochrome</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="resetPalettePushButton">
         <property name="text">
          <string>Reset Palette</string>
         </property>
         <property name="icon">
          <iconset resource="../../resource.qrc">
           <normaloff>:/resources/style.svg</normaloff>:/resources/style.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>This palette is for internal emulation purposes only.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSlider" name="redHorizontalSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="greenHorizontalSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSlider" name="blueHorizontalSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
